File: src/test/java/com/polykhel/billnext/service/mapper/WalletMapperTest.java
package com.polykhel.billnext.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class WalletMapperTest {

    private WalletMapper walletMapper;

    @BeforeEach
    public void setUp() {
        walletMapper = new WalletMapperImpl();
    }
}

File: src/test/kotlin/com/polykhel/billnext/service/mapper/WalletMapperTest.kt
package com.polykhel.billnext.service.mapper

import org.junit.jupiter.api.BeforeEach

class WalletMapperTest {

    private lateinit var walletMapper: WalletMapper

    @BeforeEach
    fun setUp() {
        walletMapper = WalletMapperImpl()
    }
}
